packages/design-system/src/figma-tokens/README.md

```diff
@@ -0,0 +1,23 @@
+# Figma Design Tokens
```

Comment:
I am not sure if keeping the file here is better than having a docs folder in the root, for discoverability purposes

Reply:
Moved to `packages/design-system/documentation/figma-design-tokens.md`